diff --git a/dist/@vitest/browser/client/__vitest_browser__/orchestrator-DM4mHHP0.js b/dist/@vitest/browser/client/__vitest_browser__/orchestrator-DM4mHHP0.js
index fddfafcde4378de80cabb3e205e3606e3dd94834..d5f463a035b22be0604bc843440bd1956d6886ed 100644
--- a/dist/@vitest/browser/client/__vitest_browser__/orchestrator-DM4mHHP0.js
+++ b/dist/@vitest/browser/client/__vitest_browser__/orchestrator-DM4mHHP0.js
@@ -306,7 +306,7 @@ If you are using a framework that manipulates browser history (like React Router
       this.iframeEvents.set(iframe, events);
     }
     events.add(event.event);
-    channel.postMessage(event);
+    setTimeout(() => channel.postMessage(event), 50);
     return new Promise((resolve, reject) => {
       const cleanupEvents = () => {
         channel.removeEventListener("message", onReceived);
diff --git a/dist/client/__vitest_browser__/orchestrator-DM4mHHP0.js b/dist/client/__vitest_browser__/orchestrator-DM4mHHP0.js
index 355b072284b0967d4a3f98362863f2b040b00c77..8240ce6c3371615d59ef04f04c5c1e0731cb08f0 100644
--- a/dist/client/__vitest_browser__/orchestrator-DM4mHHP0.js
+++ b/dist/client/__vitest_browser__/orchestrator-DM4mHHP0.js
@@ -306,7 +306,7 @@ If you are using a framework that manipulates browser history (like React Router
       this.iframeEvents.set(iframe, events);
     }
     events.add(event.event);
-    channel.postMessage(event);
+    setTimeout(() => channel.postMessage(event), 50);
     return new Promise((resolve, reject) => {
       const cleanupEvents = () => {
         channel.removeEventListener("message", onReceived);
